cmd/zstream/zstream_dump.c

```diff
@@ -385,6 +385,7 @@ zstream_do_dump(int argc, char *argv[])
 				nvlist_t *nv;
 				int sz = drr->drr_payloadlen;
 
+				(void) printf("\tnvlist_size = %d\n", sz);
```

Comment:
Maybe we should print the same name as the struct field name, as we do above (e.g. `drr_toname` -> `toname = %s`)
```suggestion
				(void) printf("\tpayloadlen = %d\n", sz);
```

Reply:
Agreed